Dmytro_Skorobohatskyi/batch_1/task7.py

```diff
@@ -0,0 +1,16 @@
+str = input('enter the string\n')
+
+answer = ""
+if len(str) == 0:
+    answer = "Fine. Be that way!"
+elif str.endswith("?"):
+    if str.isupper():
```

Comment:
Try to combine these two checks into one if statement

Reply:
look next code please
it will be as next if i combine:
elif string.endswith("?") and string.isupper():
    answer = "Calm down, I know what I'm doing!"
elif string.endswith("?") and not string.isupper():
        answer = "Sure."